WebDriverAgentLib/Categories/XCUIElement+FBTap.m

```diff
@@ -26,13 +27,35 @@ @implementation XCUIElement (FBTap)
 
 - (BOOL)fb_tapWithError:(NSError **)error
 {
+  if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"10.0") && [self fb_hasInvertedParentWindow]) {
+    // This indicates the fact, that the element is not affected by XCTest landscape bug and
+    // is properly clicked by calling the native tap selector
+    [self tap];
```

Comment:
The whole reason of having `fb_tapWithError` is to detect tap failures which are not reported by `tap`. Couldn't you simply alter hitpoint calculation here?